colors/mobilephone.lua

```diff
@@ -0,0 +1,43 @@
+ -- generated using lite-xl designer
+local style = require "core.style"
+local common = require "core.common"
+
+style.background = { common.color "#333a4a" }  -- Docview
+style.background2 = { common.color "#2f4354" } -- Treeview
+style.background3 = { common.color "#1e1f22" } -- Command view
+style.text = { common.color "#b3b3b3" }
+style.caret = { common.color "#ffffff" }
+style.accent = { common.color "#919191" }
+style.dim = { common.color "#949494" }
+style.divider = { common.color "#1e1e1e" } -- Line between nodes
+style.selection = { common.color "rgba(255,255,255,0.1)" }
```

Comment:
What do you think about using `#525b64` instead, to avoid having to deal with transparency?